staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width.go

```diff
@@ -84,10 +88,10 @@ func (e WorkEstimatorFunc) EstimateWork(r *http.Request, flowSchemaName, priorit
 }
 
 type workEstimator struct {
-	// the minimum number of seats a request must occupy
-	minimumSeats uint64
-	// the maximum number of seats a request can occupy
-	maximumSeats uint64
+	// the default minimum number of seats a request must occupy
```

Comment:
I do not see a reason to add the word "default" here, nor add "Limit" to the field name.